### PR TITLE
Rename deno prefetch to deno fetch

### LINKED ISSUE #2206: Rename "deno prefetch" to "deno fetch"
cc @bartlomieju

### PR DESCRIPTION
**Pull request by bartlomieju**
Closes #2206

### CODE REVIEW

File: tools/prefetch_test.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def prefetch_test(deno_exe):
     deno_dir = mkdtemp()
     try:
         t = os.path.join(tests_path, "006_url_imports.ts")
-        output = run_output([deno_exe, "prefetch", t],
+        output = run_output([deno_exe, "fetch", t],
```
Comment (ry) [inline tools/prefetch_test.py]: Could you also rename the file `tools/prefetch_test.py` to `tools/fetch_test.py`?

Comment (bartlomieju) [reply tools/prefetch_test.py]: Sure, no problem.

**Comment by bartlomieju** [comment]
@ry do you want `deno help` and `deno version` in this PR or should I make a separate one?

**Comment by ry** [comment]
@bartlomieju separate PR please

**Pull request merged**